plugins/generator-1.18.2/forge-1.18.2/templates/gui/gui_msg_button.java.ftl

```diff
@@ -80,13 +80,15 @@ package ${package}.network;
 			return;
 
 		<#assign btid = 0>
-        <#list data.getComponentsOfType("Button") as component>
+        <#list data.components as component>

```

Comment:
This and other files still seem unrelated to this PR to me

Reply:
Again? I removed them 😞

Reply:
The merge commit resurrected these changes...

Reply:
yeah, because I reverted the commit that added those changes and when I checked the changed files again, they were gone...

Reply:
It should be good now. I copied the files from the repository (on from the website) and they are removed.
And here is my proof if it happens again 
![image](https://user-images.githubusercontent.com/38427877/208974595-5e52e46d-ce6f-449d-a248-76f0bf17602e.png)